src/amber/server/server.cr

```diff
@@ -74,7 +74,7 @@ module Amber
           server.listen(settings.port_reuse)
           exit
         rescue e : Errno
-          if e.message == "accept: Too many open files"
+          if e.errno == Errno::EMFILE
```

Comment:
Did you see my question about whether `rescue e : Errno::EMFILE` would work so we wouldn't need to have an if statement?

Reply:
@elorest Yes, I would think it wouldn't work because Errno::EMFILE is a constant, not a class?

Reply:
Yeah looks like it returns an Int32. Have you tested this?

Reply:
@elorest Yes, I've brought down the limit with 'ulimit -n' which got the condition to trigger and it started doing server restart in a loop.